src/viur/core/email.py

```diff
@@ -2,9 +2,12 @@
 import json
 import logging
 import os
+import sys
 from abc import ABC, abstractmethod
 import typing as t
 from urllib import request
+from base64 import b64encode
```

Comment:
`base64` was already imported above, therefore its not needed to extra-import `b64encode`. See correction of b64encode-call below as well.
```suggestion
```